src/applications/edu-benefits/0994/containers/IntroductionPage.jsx

```diff
@@ -17,15 +19,18 @@ export class IntroductionPage extends React.Component {
           Equal to VA Form 22-0994 Application for Veteran Employment Through
           Technology Education Courses (VET TEC)
         </p>
-        <SaveInProgressIntro
-          startMessageOnly
-          prefillEnabled={this.props.route.formConfig.prefillEnabled}
-          verifyRequiredPrefill={
-            this.props.route.formConfig.verifyRequiredPrefill
-          }
-          messages={this.props.route.formConfig.savedFormMessages}
-          pageList={this.props.route.pageList}
-        />
+        <CallToActionWidget appId="vet-tec">
+          <SaveInProgressIntro
+            {...this.props}
```

Comment:
Is there another PR that we can reference that demonstrates how these are being used?

Reply:
Not that I know of, and honestly, we weren't using that before. I don't think this line is needed for this. It's only passing like 2 additional params that don't seem to be used.